contracts/src/nft-contracts-tzip16.ts

```diff
@@ -37,8 +37,10 @@ export async function originateNft(
         'nft_asset_main',
         'fa2_multi_nft_asset_tzip16_compat.tz'
     );
-    const storage = `(Pair (Pair (Pair "${admin}" False) None)
-  (Pair (Pair {} 0) (Pair {} {})))`;
+    const storage = `(Pair (Pair (Pair (Pair "${admin}" True) None)
+            (Pair (Pair {} 0) (Pair {} {})))
+      { Elt "description" 0x05010000000c73616d706c655f746f6b656e ;
```

Comment:
Does not look like correct TZIP-16 metadata map format.
It should have and entry with the key "" pointing to metadata uri (for the metadata in the same big_map the value should be byte-encoded "tezos-storage:content") and a second key ("content") which pints to byte-encoded JSON containing all metadata fields inside.
Consider specifying plain metadata JSON first and then encode it. This way it is possible to verify the content from reading source code